src/test/java/educationalproject/programmingstuff/tests_integrations/EndpointsProcessingTests.java

```diff
@@ -0,0 +1,171 @@
+package educationalproject.programmingstuff.tests_integrations;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import educationalproject.programmingstuff.TestDataCreator;
+import educationalproject.programmingstuff.model.User;
+import educationalproject.programmingstuff.repositories.ItemRepository;
+import educationalproject.programmingstuff.repositories.OrderRepository;
+import educationalproject.programmingstuff.repositories.UserRepository;
+import educationalproject.programmingstuff.servicies.UserService;
+import educationalproject.programmingstuff.servicies.dto.UserCreateRequestDto;
+import educationalproject.programmingstuff.servicies.dto.UserResponseDto;
+import educationalproject.programmingstuff.servicies.mappers.UserResponseMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.RequestBuilder;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class EndpointsProcessingTests {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ItemRepository itemRepository;
+
+    @Autowired
+    private OrderRepository orderRepository;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private UserResponseMapper userResponseMapper;
+
+    @Autowired
+    private UserService userService;
+
+    private RequestBuilder request;
+
+    private List<User> userEntities;
+
+    private List<UserResponseDto> userResponseDtos;
+
+    private String expectedResponse;
```

Comment:
it's a bad idea to have the common state in tests, because each test should be independent
use local variables in each test